jenkins/job-configs/kubernetes-jenkins-pull/bootstrap-pull.yaml

```diff
@@ -162,15 +162,15 @@
     - kubernetes-node-e2e:
         max-total: 12
         job-name: pull-kubernetes-node-e2e
-        json: 0
+        json: 1
```

Comment:
why not use true/false here?

if these fields will all go away soon, that's fine too.